src/main/java/org/ladocuploader/app/cli/TransmitterCommands.java

```diff
@@ -61,28 +68,24 @@ public void transmit() throws IOException, JSchException, SftpException {
     }
 
     private void transmitBatch(List<Submission> submissions, TransmissionType transmissionType) throws IOException, JSchException, SftpException{
-        String zipFilename = createZipFilename(transmissionType);
-        // TODO: get errors for the packageType
+
+        UUID runId = UUID.randomUUID();
+        String zipFilename = createZipFilename(transmissionType, runId);
         Map<String, Object> zipResults = zipFiles(submissions, zipFilename);
 
         List<UUID> successfullySubmittedIds = (List<UUID>) zipResults.get("success");
 
         Map<UUID, Map<CsvType, String>> failedSubmissions = (Map<UUID, Map<CsvType, String>>) zipResults.get("failed");
 
+        Map<UUID, Map<String, String>> failedDocumentation = (Map<UUID, Map<String, String>>) zipResults.get("failed_documentation");
```

Comment:
Might be nice to have these 'failed_documentation' keys as variables since theyre referenced in more than one place

Reply:
Done